fineract-loan/src/main/java/org/apache/fineract/portfolio/loanproduct/api/LoanProductsApiResourceSwagger.java

```diff
@@ -1402,6 +1366,9 @@ private GetLoanCharge() {}
         @Schema(example = "[]")
         public List<Integer> numberOfRepaymentVariationsForBorrowerCycle;
         public GetLoanProductsResponse.GetLoanProductsAccountingRule accountingRule;
+        public StringEnumOptionData daysInMonthType;
+        public StringEnumOptionData daysInYearType;
+        public StringEnumOptionData daysInYearCustomStrategy;
```

Comment:
Only `daysInYearCustomStrategy` is String enum option.. the `daysInMonthType` and `daysInYearType` are `EnumOptionData` please modify them back!